kernel/Makefile

```diff
@@ -149,9 +149,9 @@ show-tags: show-tag_$(2)$(3)$(4)
 # with DCT as part of the dependency on build_$(2)$(3)$(4) and then build
 # with DOCKER_CONTENT_TRUST explicitly set to 0
 
-# Skip perf build for now. See:
-# https://github.com/linuxkit/linuxkit/issues/3299
-ifeq ($(ARCH),SKIP)
+# Only build perf on latest LTS and stable kernels.  Skip arm64.
+ifeq ($(2), $(filter $(2),5.3.x 4.19.x))
+ifneq ($(ARCH),aarch64)
```

Comment:
Are there known issues on arm64 build?

Reply:
I don't have an arm64 rig to test on, so I can't comment.  I'm merely reverting the fix that disabled `perf` entirely.  Prior to that, it was disabled for the arm64 build.  See here: https://github.com/linuxkit/linuxkit/commit/94cdfb55b62fbeb4829ad9575116d4d29e26186a#diff-74d64d85ef940cb8fbcd3540a1f97c9c

Reply:
@zlim I can't remember if I originally tried on arm64 or not, but don't have the cycles at the moment to test bcc on arm64.